x/programs/rust/wasmlanche-sdk/src/lib.rs

```diff
@@ -5,6 +5,7 @@ pub mod types;
 
 mod logging;
 mod memory;
+pub mod panic;
```

Comment:
Maybe make the module name, `debug`?

Reply:
Just removed it and put the function in the `logging` module instead.